components/src/core/ChannelValue/ChannelValue.tsx

```diff
@@ -65,17 +73,17 @@ export const ChannelValue: React.FC<ChannelValueProps> = (props) => {
         });
 
     return (
-        <span className={classes.wrapper} style={{ fontSize, color }} data-test={'wrapper'}>
+        <span className={clsx(defaultClasses.root, classes.root)} style={{ fontSize, color }} data-test={'wrapper'}>
             {icon && (
-                <span className={classes.icon} data-test={'icon'}>
+                <span className={clsx(defaultClasses.icon, classes.icon)} data-test={'icon'}>
                     {changeIconDisplay(icon)}
                 </span>
             )}
             {prefix && getUnitElement()}
             <Typography
                 variant={'h6'}
                 color={'inherit'}
-                className={combine([classes.text, classes.value])}
+                className={clsx(combine([defaultClasses.text, defaultClasses.value]), classes.value)}
```

Comment:
I think with clsx here, you can drop the combine function

Reply:
I dropped the use of combine, seems to look the same.